### ISSUE TITLE
关于start_logits和end_logits的疑问

### ISSUE THREAD

**Issue by xuanzebi** [ShannonAI/mrc-for-flat-nested-ner#57]
Trainer.py 代码241行
![image](https://user-images.githubusercontent.com/26642184/96530620-6608c580-12ba-11eb-989a-ff56f1bd668d.png)
得到start_logits和end_logits后，这两个是没有过softmax/sigmoid的吧，这个时候不应该过一下sigmoid函数再然后和0.5比较得到二分类的结果吗？

**Comment by YuxianMeng** [comment]
@xuanzebi x>=0 is equivalent to sigmoid(x)>=0.5

**Comment by xuanzebi** [comment]
好吧 是等价的两个
